src/hubmap_translation/addl_index_transformations/portal/add_assay_details.py

```diff
@@ -22,8 +22,9 @@ def _get_assay_details(doc, transformation_resources):
         raise
     json = response.json()
     if not json:
-        logger.info(f"No soft assay information returned for dataset ${uuid}.")
-        return {'assaytype': 'unknown', 'description': dataset_type, 'vitessce-hints': ['unknown-assay']}
+        empty_error_msg = 'No soft assay information returned.'
+        logger.info(f"${empty_error_msg} For dataset ${uuid}.")
+        return {'description': dataset_type, 'vitessce-hints': ['unknown-assay'], 'error': empty_error_msg}
     return json
```

Comment:
Lines 23-28 should be moved inside the try block as response.json() will raise a [JSONDecodeError](https://requests.readthedocs.io/en/latest/api/#requests.JSONDecodeError) if the response format is not valid json.

Reply:
`{}` shouldn't raise an error.